views/js/controller/tests/action.js

```diff
@@ -31,13 +31,16 @@ define([
     const logger = loggerFactory('taoTests/controller/action');
 
     binder.register('testPreview', function testPreview(actionContext) {
-        previewerFactory(
-            module.config().provider,
+        const config = module.config();
+        const previewerConfig = _.omit({
+            readOnly: false,
+            fullPage: true,
+            pluginsOptions: config.pluginsOptions
```

Comment:
FYI: this is wrong, didn't notice that this module is applied for delivery too, not just for previewer. Module config  must be moved to another place.